### PR TITLE
Added Simplified Chinese

### PR DESCRIPTION
**Pull request by seanzhang98**
#### To-Dos

- [x] Successful build `yarn build`
- [ ] Translation keys `yarn i18n:extract`
- [ ] Database migration (if required)

#### Issues Fixed or Closed

- Fixes #XXXX

### CODE REVIEW

File: src/i18n/locale/zh_Hans.json [hunk 1]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.generalsettingsDescription": "Overseerr 的全局和默认设置。",
+  "components.Settings.notificationsettingsDescription": "关于通知的全局设置。",
+  "components.Settings.notificationAgentSettingsDescription": "设置通知类型和代理服务。",
+  "components.Settings.Notifications.emailNotificationTypesAlertDescription": "「请求提交」、「请求失败」这两种通知类型，仅发送通知給有「请求管理」权限的用户。",
+  "components.UserProfile.UserSettings.UserGeneralSettings.regionTip": "以地区筛选结果（仅于「热门」、「即将上映」）",
+  "components.Settings.regionTip": "以地区筛选结果（仅于「热门」、「即将上映」）",
+  "components.UserProfile.UserSettings.UserGeneralSettings.originallanguageTip": "以原始语言筛选结果（仅于「热门」、「即将上映」）",
+  "components.Settings.originallanguageTip": "以原始语言筛选结果（仅于「热门」、「即将上映」）",
+  "components.Settings.SettingsJobsCache.jobsDescription": "Overseerr 将定时运行以下的維護任務。手动运行工作不会影響它正常的时间表。",
+  "components.Settings.plexsettingsDescription": "关于 Plex 服务器的设置。Overseerr 将定时运行媒体库扫描。",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.SettingsJobsCache.jobsDescription": "Jellyseerr 将定时运行以下的維護任務。手动运行工作不会影響它正常的时间表。",

File: src/i18n/locale/zh_Hans.json [hunk 2]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.Notifications.smtpHost": "SMTP 主机",
+  "i18n.partiallyavailable": "部分可观看",
+  "i18n.unavailable": "不可观看",
+  "components.TvDetails.unavailable": "不可观看",
+  "components.StatusChacker.newversionavailable": "软件可更新",
+  "components.Layout.alphawarning": "这是预览版软件，所以可能会不穩定或被破坏。请向 GitHub 反馈问题！",
+  "components.TitleCard.tvshow": "电视节目",
+  "components.StatusBadge.status4k": "4K 版 {status}",
+  "components.Setup.tip": "提示",
+  "components.Setup.welcome": "欢迎來到 Overseerr",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Setup.welcome": "欢迎來到 Jellyseerr",

File: src/i18n/locale/zh_Hans.json [hunk 3]
```diff
@@ -0,0 +1,811 @@
...
+  "components.UserEdit.email": "电子邮件地址",
+  "components.UserEdit.settings": "设置管理",
+  "components.UserList.admin": "管理员",
+  "components.UserEdit.admin": "管理员权限",
+  "components.TvDetails.userrating": "评分",
+  "components.TvDetails.status": "状态",
+  "components.Settings.SonarrModal.baseUrl": "网站根目录",
+  "components.Settings.RadarrModal.baseUrl": "网站根目录",
+  "components.Settings.RadarrModal.apiKeyPlaceholder": "您的 Radarr 应用程序密钥",
+  "components.StatusChacker.reloadOverseerr": "重新启动 Overseerr",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.StatusChacker.reloadOverseerr": "重新启动 Jellyseerr",

File: src/i18n/locale/zh_Hans.json [hunk 4]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.runnow": "运行",
+  "components.Settings.notrunning": "未运行",
+  "components.Settings.nextexecution": "下一次运行时间",
+  "components.Settings.activeProfile": "現行质量设置",
+  "components.Settings.notificationsettings": "通知设置",
+  "components.Settings.default4k": "设置 4K 为默认分辨率",
+  "components.Settings.currentlibrary": "当前媒体库： {name}",
+  "components.Settings.SonarrModal.seasonfolders": "季数文件夹",
+  "components.Settings.SettingsAbout.overseerrinformation": "Overseerr 信息",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.SettingsAbout.overseerrinformation": "Jellyseerr 信息",

File: src/i18n/locale/zh_Hans.json [hunk 5]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.RadarrModal.testFirstRootFolders": "请先测试连接",
+  "components.Settings.RadarrModal.testFirstQualityProfiles": "请先测试连接",
+  "components.Settings.SonarrModal.toastRadarrTestSuccess": "Sonarr 服务器连接成功！",
+  "components.Settings.SonarrModal.defaultserver": "默认服务器",
+  "components.Settings.nodefault": "没有默认服务器",
+  "components.Settings.deleteserverconfirm": "确定要删除这个服务器吗？",
+  "components.Settings.addradarr": "添加 Radarr 服务器",
+  "components.Settings.addsonarr": "添加 Sonarr 服务器",
+  "components.Settings.SonarrModal.server4k": "4K 服务器",
+  "components.Settings.SettingsAbout.supportoverseerr": "支持 Overseerr",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.SettingsAbout.supportoverseerr": "支持 Jellyseerr",

File: src/i18n/locale/zh_Hans.json [hunk 6]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.SonarrModal.ssl": "安全通信协议（SSL）",
+  "components.Settings.RadarrModal.ssl": "安全通信协议（SSL）",
+  "components.Settings.port": "端口",
+  "components.Settings.SonarrModal.port": "端口",
+  "components.Settings.RadarrModal.port": "端口",
+  "components.Settings.Notifications.NotificationsPushover.userToken": "用户令牌",
+  "components.Settings.Notifications.NotificationsPushover.accessToken": "应用程序 API 令牌",
+  "components.Settings.menuNotifications": "通知",
+  "components.Settings.menuLogs": "日志",
+  "components.Settings.menuAbout": "关于 Overseerr",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.menuAbout": "关于 Jellyseerr",

File: src/i18n/locale/zh_Hans.json [hunk 7]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.SettingsAbout.documentation": "文件",
+  "components.UserEdit.voteDescription": "允许投票（功能尚未实现）",
+  "components.Settings.SonarrModal.baseUrlPlaceholder": "例：/sonarr",
+  "components.Settings.RadarrModal.baseUrlPlaceholder": "例：/radarr",
+  "components.RequestModal.pending4krequest": "{title} 的 4K 请求",
+  "components.RequestModal.pendingrequest": "{title} 的请求",
+  "components.RequestModal.extras": "特辑",
+  "components.RequestList.RequestItem.notavailable": "N/A",
+  "components.Settings.SettingsAbout.Releases.versionChangelog": "更改日志",
+  "components.Settings.SettingsAbout.Releases.runningDevelop": "您正在使用 Overseerr 的开发版本！",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.SettingsAbout.Releases.runningDevelop": "您正在使用 Jellyseerr 的开发版本！",

File: src/i18n/locale/zh_Hans.json [hunk 8]
```diff
@@ -0,0 +1,811 @@
...
+  "components.TvDetails.watchtrailer": "观看预告片",
+  "components.TvDetails.showtype": "节目类型",
+  "components.TvDetails.similar": "类似",
+  "components.TvDetails.similarsubtext": "类似 {title} 的电视节目",
+  "components.UserEdit.avatar": "头像",
+  "components.UserEdit.request4kTv": "提交 4K 电视节目的请求",
+  "components.RequestModal.requestfrom": "{username} 的请求待处理",
+  "components.RequestModal.request4kfrom": "{username} 的 4K 请求待处理。",
+  "components.Settings.toastApiKeySuccess": "生成新应用程序密钥成功！",
+  "components.Settings.SettingsAbout.clickheretojoindiscord": "加入 Overseerr 的 Discord 服务器！",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.SettingsAbout.clickheretojoindiscord": "加入 Jellyseerr 的 Discord 服务器！",

File: src/i18n/locale/zh_Hans.json [hunk 9]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.SonarrModal.servernamePlaceholder": "Sonarr 服务器",
+  "components.Settings.SonarrModal.servername": "服务器名称",
+  "components.Settings.SonarrModal.editsonarr": "编辑 Sonarr 服务器",
+  "components.Settings.SonarrModal.add": "添加服务器",
+  "components.Settings.RadarrModal.servernamePlaceholder": "Radarr 服务器",
+  "components.Settings.RadarrModal.servername": "服务器名称",
+  "components.Settings.RadarrModal.editradarr": "编辑 Radarr 服务器",
+  "components.Settings.RadarrModal.defaultserver": "默认服务器",
+  "components.Settings.RadarrModal.add": "添加服务器",
+  "components.StatusChacker.newversionDescription": "Overseerr 软件可更新。请点击以下的按钮重新启动。",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.StatusChacker.newversionDescription": "Jellyseerr 软件可更新。请点击以下的按钮重新启动。",

File: src/i18n/locale/zh_Hans.json [hunk 10]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.localLoginTip": "禁用此设置只会阻止新登陆（用户不会被删除）",
+  "components.Settings.csrfProtectionHoverTip": "除非您了解此功能，请勿启用它！",
+  "components.UserList.users": "用户",
+  "components.Settings.settings": "设置",
+  "components.Settings.applicationTitle": "应用程序名",
+  "components.Search.search": "搜索",
+  "components.Setup.setup": "配置",
+  "components.Discover.discover": "探索",
+  "components.AppDataWarning.dockerVolumeMissing": "无法访问 Docker 目录",
+  "components.AppDataWarning.dockerVolumeMissingDescription": "必须使用目录（bind mount）指定某个宿主机器的资料夹跟容器內的 <code>{appDataPath}</code> 资料夹連通，才能保存 Overseerr 的配置和数据。",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.AppDataWarning.dockerVolumeMissingDescription": "必须使用目录（bind mount）指定某个宿主机器的资料夹跟容器內的 <code>{appDataPath}</code> 资料夹連通，才能保存 Jellyseerr 的配置和数据。",

File: src/i18n/locale/zh_Hans.json [hunk 11]
```diff
@@ -0,0 +1,811 @@
...
+  "components.PermissionEdit.voteDescription": "为请求投票的权限。",
+  "components.RequestModal.AdvancedRequester.loadinglanguages": "载入中…",
+  "components.Settings.SonarrModal.validationLanguageProfileRequired": "必须设置语言",
+  "components.Settings.SonarrModal.testFirstLanguageProfiles": "请先测试连接",
+  "components.Settings.SonarrModal.selectLanguageProfile": "语言设置",
+  "components.Settings.SonarrModal.loadinglanguageprofiles": "载入中…",
+  "components.Settings.SonarrModal.languageprofile": "语言设置",
+  "components.Settings.SonarrModal.animelanguageprofile": "动漫语言设置",
+  "components.RequestModal.AdvancedRequester.languageprofile": "语言设置",
+  "components.PermissionEdit.settingsDescription": "授予管理所有 Overseerr 设置的权限。",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.PermissionEdit.settingsDescription": "授予管理所有 Jellyseerr 设置的权限。",

File: src/i18n/locale/zh_Hans.json [hunk 12]
```diff
@@ -0,0 +1,811 @@
...
+  "components.RequestModal.SearchByNameModal.notvdbiddescription": "无法自动配对您的请求。请从以下列表中选择正確的媒体項：",
+  "components.CollectionDetails.requestswillbecreated4k": "为以下的电影提交 4K 请求：",
+  "components.CollectionDetails.requestcollection4k": "提交 4K 系列请求",
+  "components.CollectionDetails.request4k": "提交 4K 请求",
+  "components.Settings.SettingsAbout.Releases.runningDevelopMessage": "开发版本的更改日志不会顯示在以下。请直接到 <GithubLink>GitHub</GithubLink> 查看更改日志。",
+  "components.Settings.Notifications.settinguptelegramDescription": "要设置 Telegram 通知，必须先<CreateBotLink>生成一个 「bot」 机器人</CreateBotLink>以及把 <GetIdBotLink>@get_id_bot</GetIdBotLink>加到聊天室，然后在以下输入机器人应用程序密钥和聊天室的 ID 號。",
+  "components.Settings.Notifications.NotificationsPushbullet.settingUpPushbulletDescription": "要设置 Pushbullet 通知，必须先<CreateAccessTokenLink>获取 API 令牌</CreateAccessTokenLink>，然后在下面输入。",
+  "components.Settings.Notifications.NotificationsPushover.settinguppushoverDescription": "要设置 Pushover 通知，必须先<RegisterApplicationLink>生成一个应用程序</RegisterApplicationLink>，然后在以下输入生成的 API 令牌和您的用户密钥。（添加應用时可以使用 <IconLink>Overseerr 的官方圖标</IconLink>。）",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.Notifications.NotificationsPushover.settinguppushoverDescription": "要设置 Pushover 通知，必须先<RegisterApplicationLink>生成一个应用程序</RegisterApplicationLink>，然后在以下输入生成的 API 令牌和您的用户密钥。（添加應用时可以使用 <IconLink>Jellyseerr 的官方圖标</IconLink>。）",

File: src/i18n/locale/zh_Hans.json [hunk 13]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.Notifications.NotificationsSlack.settingupslackDescription": "要设置 Slack 通知，必须先<WebhookLink>生成一个 「incoming webhook」</WebhookLink>，然后在以下输入生成的Weebhook网址。",
+  "components.Settings.trustProxyTip": "使用代理服务器时，允许 Overseerr 搜索客戶端 IP 位址（Overseerr 必须重新启动）",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.trustProxyTip": "使用代理服务器时，允许 Jellyseerr 搜索客戶端 IP 位址（Jellyseerr 必须重新启动）",

File: src/i18n/locale/zh_Hans.json [hunk 14]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.csrfProtectionTip": "设置外部访问权限为只讀（Overseerr 必须重新启动）",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.csrfProtectionTip": "设置外部访问权限为只讀（Jellyseerr 必须重新启动）",

File: src/i18n/locale/zh_Hans.json [hunk 15]
```diff
@@ -0,0 +1,811 @@
...
+  "components.RegionSelector.regionDefault": "所有地区",
+  "components.UserProfile.UserSettings.UserGeneralSettings.region": "「探索」地区",
+  "components.Settings.region": "「探索」地区",
+  "components.UserProfile.UserSettings.UserGeneralSettings.originallanguage": "「探索」语言",
+  "components.Settings.originallanguage": "「探索」语言",
+  "components.Discover.upcomingtv": "即将上映的电视节目",
+  "components.Settings.webhook": "Weebhook",
+  "components.Settings.email": "电子邮件",
+  "components.Settings.Notifications.emailNotificationTypesAlert": "电子邮件通知收件人",
+  "components.Settings.generalsettingsDescription": "Overseerr 的全局和默认设置。",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.generalsettingsDescription": "Jellyseerr 的全局和默认设置。",

File: src/i18n/locale/zh_Hans.json [hunk 16]
```diff
@@ -0,0 +1,811 @@
...
+  "components.Settings.notificationsettingsDescription": "关于通知的全局设置。",
+  "components.Settings.notificationAgentSettingsDescription": "设置通知类型和代理服务。",
+  "components.Settings.Notifications.emailNotificationTypesAlertDescription": "「请求提交」、「请求失败」这两种通知类型，仅发送通知給有「请求管理」权限的用户。",
+  "components.UserProfile.UserSettings.UserGeneralSettings.regionTip": "以地区筛选结果（仅于「热门」、「即将上映」）",
+  "components.Settings.regionTip": "以地区筛选结果（仅于「热门」、「即将上映」）",
+  "components.UserProfile.UserSettings.UserGeneralSettings.originallanguageTip": "以原始语言筛选结果（仅于「热门」、「即将上映」）",
+  "components.Settings.originallanguageTip": "以原始语言筛选结果（仅于「热门」、「即将上映」）",
+  "components.Settings.SettingsJobsCache.jobsDescription": "Overseerr 将定时运行以下的維護任務。手动运行工作不会影響它正常的时间表。",
```
Comment (fallenbagel) [inline src/i18n/locale/zh_Hans.json]: "components.Settings.SettingsJobsCache.jobsDescription": "Jellyseerr 将定时运行以下的維護任務。手动运行工作不会影響它正常的时间表。",

**Review by Fallenbagel — CHANGES_REQUESTED** [review]
Please replace the `overseerr` with `jellyseerr`

**Comment by seanzhang98** [comment]
updated

**Comment by Fallenbagel** [comment]
@all-contributors please add @seanzhang98  for translation and code

**Comment by Fallenbagel** [comment]
@all-contributors please add @seanzhang98 for translation and code

**Comment by Fallenbagel** [comment]
@all-contributors please add @seanzhang98 for translation and code

**Comment by Fallenbagel** [comment]
@all-contributors please add @seanzhang98 for translation and code

**Comment by Fallenbagel** [comment]
@all-contributors please add @seanzhang98 for translation and code

**Comment by Fallenbagel** [comment]
@all-contributors please add @seanzhang98 for translation and code

**Pull request merged**